source/Loggly/Transports/HttpTransports/HttpMessageTransport.cs

```diff
@@ -29,7 +29,6 @@ internal HttpMessageTransport(HttpMessageHandler messageHandler)
         {
             HttpClient = new HttpClient(messageHandler);
             HttpClient.DefaultRequestHeaders.UserAgent.ParseAdd(_userAgent);
-            HttpClient.DefaultRequestHeaders.Add("Connection", "close");
```

Comment:
Adding this header does not allow us to reuse the connection. A POST request without any header against Loggly results in a return header of `keep-alive` so that seems to be the default behavior. When `close` is specified in the request, `close` is also returned in the response. Removing this line will allow us to use reuse connections

Reply:
Thanks for the contribution!

@snakefoot will merge in 24 hrs unless you have any comment?

Reply:
Don't know much about Loggly-protocol, so have no opinion :)

Reply:
Awesome! When we get this in, I can also make a PR to bring it into [nlog-targets-loggly](https://github.com/neutmute/nlog-targets-loggly)

Reply:
@neutmute @snakefoot Can we please bring this in if there are no more comments/concerns? Would love to see the effects of this change

Reply:
@amikapoor I was hoping you had tested the code-change already, by compiling a build using the code in the pull-request. If you are just making "random" changes, then maybe you should implement a setting, where the code-change is default turned off. Anyway not my decision.

Reply:
I haven't tested the code change in my deployed Function App on Azure where I'm seeing issues caused by this line due to the repeated opening and closing of connections because of this header. However, I have tested the effects of this change using Postman and by researching. If you post without specifying a header, the response from Loggly says to `keep-alive`. Adding this header overrides the default behavior and closes the connection after each log statement. The goal of this change is to use the default connection behavior of Loggly's protocol, which is `keep-alive`. [This article](https://www.imperva.com/learn/performance/http-keep-alive/) is a good read too

Reply:
will merge but agree with snakefoot